cmd/waku/node.go

```diff
@@ -534,6 +537,37 @@ func loadPrivateKeyFromFile(path string, passwd string) (*ecdsa.PrivateKey, erro
 	return crypto.ToECDSA(pKey)
 }
 
+/*
```

Comment:
What do you think if we instead remove this function completely and not only leave it commented. We can always rely on commit history to restore it back.